include/seqan3/alignment/pairwise/edit_distance_fwd.hpp

```diff
@@ -0,0 +1,68 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Forwards for seqan3::edit_distance_unbanded related types.
+ * \author Marcel Ehrhardt <marcel.ehrhardt AT fu-berlin.de>
+*/
+
+#pragma once
+
+#include <seqan3/core/platform.hpp>
+#include <seqan3/std/concepts>
+#include <seqan3/std/ranges>
+
+namespace seqan3::detail
+{
+/*!\todo Document me
```

Comment:
there is still a todo :eyes:

Reply:
Yes, I also saw this, but it's copy/paste ... so I don't think it's necessarily within the scope of this PR (though it would be nice)

Reply:
In general, if this is going to be part of the release we need to have it documented, although it is developer API, but I wish to have a release with no todo in the documentation. If you can promise this in a related PR to this, which will follow soon, then we can leave it.

Reply:
@rrahn This todo is from you :laughing:

Reply:
pah, everyone could claim this 😏